examples/002_beambeam/003_beambeam3d_strongstrong_track.py

```diff
@@ -81,9 +81,8 @@
 config_for_update=xf.ConfigForUpdateBeamBeamBiGaussian3D(
```

Comment:
Would it make senso to attach this function to the pipeline_manager  or to the beam-beam element